tests/unit/test-class-wc-payments.php

```diff
@@ -167,6 +167,41 @@ function () {
 		$this->assertNotEquals( 'woocommerce_rest_missing_nonce', $response->get_data()['code'] );
 	}
 
+	public function test_temp() {
+		// Necessary in order to prevent wp_die() from being called.
+		define( 'DOING_AJAX', true );
+
+		$self        = $this;
+		$customer_id = 'cus_123456789';
+
+		$pre_http_request_cb = function ( $preempt, $parsed_args, $url ) use ( $self, $customer_id ) {
+			$body = json_decode( $parsed_args['body'] );
+			$self->assertEquals( $customer_id, $body->customer_id );
```

Comment:
I also missed this yesterday but I did a quick test and found that `$self` can be removed and `$this` can be used in its place.